program/rust/src/c_oracle_header.rs

```diff
@@ -7,11 +7,32 @@ use bytemuck::{
     Pod,
     Zeroable,
 };
-
+use std::mem::size_of;
 //bindings.rs is generated by build.rs to include
 //things defined in bindings.h
 include!("../bindings.rs");
 
+pub trait PythStruct: Pod {
```

Comment:
I think `PythAccount` is maybe a better name? Could we also add a comment explaining the purpose of this trait? Thank-you :D